src/lib/OpenEXRCore/chunk.c

```diff
@@ -348,8 +330,8 @@ extract_chunk_leader (
             return ctxt->print_error (
                 ctxt,
                 EXR_ERR_BAD_CHUNK_LEADER,
-                "Invalid chunk size reconstructing chunk table: found out of range %"PRId64,
```

Comment:
the actual fix is here

https://github.com/AcademySoftwareFoundation/openexr/pull/1406